iac/terraform/aws/web-client/s3_bucket.tf

```diff
@@ -0,0 +1,105 @@
+#checkov:skip=CKV_AWS_145: "Ensure that S3 buckets are encrypted with KMS by default"
+resource "aws_s3_bucket" "static_site" {
+  #checkov:skip=CKV2_AWS_61: "Ensure that an S3 bucket has a lifecycle configuration"
+  #checkov:skip=CKV2_AWS_62: "Ensure S3 buckets should have event notifications enabled"
+  #checkov:skip=CKV_AWS_144: "Ensure that S3 bucket has cross-region replication enabled"
+  #checkov:skip=CKV_AWS_145: "Ensure that S3 buckets are encrypted with KMS by default"
+  #checkov:skip=CKV_AWS_18: "Ensure the S3 bucket has access logging enabled"
+  #checkov:skip=CKV_AWS_21: "Ensure all data stored in the S3 bucket have versioning enabled"
+  bucket = "${var.environment}-bloodconnect-web-client"
+
+  tags = {
+    Name = "StaticSiteBucket"
+  }
+}
+
+resource "aws_s3_bucket" "log_store" {
```

Comment:
Log should not be kept for unlimited time. Add an expiration please

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_lifecycle_configuration

Reply:
https://github.com/CraftsmenLtd/BloodConnect/pull/119/files#diff-127209aebb8af744a3226945bcb6d23c19c1a6cb974ee1a9a15cf8464aea34e7R28-R43